moveit_planners/trajopt/trajopt/LICENSE

```diff
@@ -0,0 +1,11 @@
+Copyright (c) 2013, John Schulman
```

Comment:
Within the two packages you are copying in, add a README that simply says something to the effect of:

> TODO(ommmid): Remove code duplicaiton
> This package temporarily copied into moveit until the repo trajopt_ros does not depend on Tesseract
> See https://github.com/ros-industrial-consortium/trajopt_ros/issues/122